File: orgservice-new/src/main/java/com/thoughtmechanix/organization/repository/OrganizationRepository.java
package com.thoughtmechanix.organization.repository;

import com.thoughtmechanix.organization.model.Organization;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface OrganizationRepository extends JpaRepository<Organization,String>{

}

File: organization-service/src/main/kotlin/com/github/hfantin/organizationservice/repository/OrganizationRepository.kt
package com.github.hfantin.organizationservice.repository

import com.github.hfantin.organizationservice.model.Organization
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface OrganizationRepository : JpaRepository<Organization, String>
